### ISSUE TITLE
Cant render header on taxonomy page

### ISSUE THREAD

**Issue by keyolk** [matcornic/hugo-theme-learn#28]
Hi I got below error while I'm trying to use taxonomy.  
```bash
ERROR: 2017/01/15 22:32:49 template.go:132: template: theme/partials/header.html:24:30: executing "theme/partials/header.html" at <.RelPerma
link>: can't evaluate field RelPermalink in type *hugolib.Node in theme/partials/header.html
ERROR: 2017/01/15 22:32:49 template.go:132: template: theme/partials/header.html:24:30: executing "theme/partials/header.html" at <.RelPerma
link>: can't evaluate field RelPermalink in type *hugolib.Node in theme/partials/header.html
```

Seems it related to
https://discuss.gohugo.io/t/cant-evaluate-field-relpermalink-in-type-hugolib-node/4280/3

Any of workaround ? 

```bash
$ hugo version
Hugo Static Site Generator v0.17 BuildDate: 2016-12-18T03:13:22-08:00
```

**Comment by matcornic** [comment]
Same than #29 . Try with v0.18, and add more information if the problem is still present.

**Comment by keyolk** [comment]
It solved with v0.18.
Thanks to quick resopnse. : )
